### ISSUE TITLE
influenza

### ISSUE THREAD

**Issue by MuYi-Yang-cell** [ablab/spades#1178]
### Is your feature request related to a problem? Please describe. For generic questions use Q&A section in the Discussions forum above.

Hello,

I'm currently using SPAdes for de novo assembly of avian influenza virus genomes. The genome consists of 8 independent segments and was sequenced using Illumina NextSeq for paired-end reads (2x100). However, the scaffold results from the assembly are not as satisfactory as I hoped. I noticed that someone else had asked a similar question before. I've tried using the following parameters for the assembly. Is this the optimal parameter combination, or do you have any other suggestions?
Thank you

python coronaspades.py --custom-hmms U-RVDBv20.0-prot-influenza.hmm.gz -t 8 -1 ../WA2308002_1.fq -2 ../WA2308002_2.fq -o Filter_SPAdes

**Comment by asl** [comment]
Can you provide the spades.log file at least?

**Comment by MuYi-Yang-cell** [comment]
Okay, here is the log file from my run. My description might have been a bit unclear. I wanted to inquire whether this parameter configuration is the optimal one. If I were to include a reference genome, would it potentially improve the assembly results?

[spades.log](https://github.com/ablab/spades/files/12482866/spades.log)

**Comment by MuYi-Yang-cell** [comment]
I apologize, this is the actual log file.

**Spades.log**

[spades.log](https://github.com/ablab/spades/files/12483003/spades.log)

**Comment by asl** [comment]
All right. Can you also upload `hmm_statistics.txt` from the output dir?

**Comment by MuYi-Yang-cell** [comment]
Sure, the file within this result folder `hmm_statistics.txt`

[hmm_statistics.txt](https://github.com/ablab/spades/files/12483061/hmm_statistics.txt)

**Comment by asl** [comment]
Ok, and `assembly_graph_with_scaffolds.gfa` as well, please

**Comment by MuYi-Yang-cell** [comment]
Right here.

[assembly_graph_with_scaffolds.txt](https://github.com/ablab/spades/files/12483146/assembly_graph_with_scaffolds.txt)

**Comment by asl** [comment]
Ok, so there are 8 segments that are almost completely assembled. Why you're saying "not as satisfactory"? What is wrong?

**Comment by MuYi-Yang-cell** [comment]
Indeed, I have reviewed the `scaffolds.fasta` results, and this is the best outcome I have achieved after multiple attempts. My description might have been a bit unclear. Is this the optimal parameter combination, or do you have any other suggestions?

**Comment by asl** [comment]
This was exactly the way how the Influenza benchmark benchmark was performed in coronaSPAdes paper.
